src/test/java/edu/oakland/test/database/DatabaseCommManagerTest.java

```diff
@@ -0,0 +1,92 @@
+package edu.oakland.test.database;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.database.DatabaseCommManagerImplementation;
+import java.lang.IllegalArgumentException;
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Database CommManager Test")
+public class DatabaseCommManagerTest {
+
+  @Test
+  @DisplayName("rfid going in is Rfid coming Out")
+  void rfidInIsRfidOut() {
+
+    int i = 10;
+
+    DatabasePersistentStorageStub dpss = new DatabasePersistentStorageStub();
+    assertEquals(i, dpss.locateRfidData());
+  }
+
+  @Test
+  @DisplayName("Offset matches Datapoint")
+  void offsetMatchDataPoint() {
+    DatabasePersistentStorageStub dpss = new DatabasePersistentStorageStub();
+
+    float lat = (float) (Math.random() * 360 - 180);
+    float lng = (float) (Math.random() * 360 - 180);
+
+    LocationDataPoint ldp = new LocationDataPoint(lat, lng,
+        LocalDateTime.of((int) (Math.random() * 50 + 1970),
+         (int) (Math.random() * 12 + 1),
+            (int) (Math.random() * 28 + 1),
+             (int) (Math.random() * 24),
+              (int) (Math.random() * 60)));
+
+    ldp = null;
```

Comment:
We shouldn't be setting what we just randomly generated to `null`. `null` is an edge case that should be handled separately, and should not be used as a default test case. Instead, let's pass our randomly generated `LocationDataPoint` to the the `DatabasePersistentStorageStub`.
```suggestion
    dpss.storeLocationDataPoint(ldp);
```